test/statetest/statetest_runner.cpp

```diff
@@ -25,6 +26,14 @@ void run_state_test(const StateTransitionTest& test, evmc::VM& vm)
             const auto tx = test.multi_tx.get(expected.indexes);
             auto state = test.pre_state;
 
+            for (const auto& [add, acc] : state.get_accounts())
```

Comment:
Add a comment for these block of code.
Enclose statements that are more than single line with `{}`.

Reply:
done